README.md

```diff
@@ -200,3 +201,68 @@ SET search_path TO api;
 ## Reporting tools that use data from logserver
 - https://github.com/uwcirg/gwen
 - https://github.com/uwcirg/logserver-tabulator
+
+## Setting up development environment
+
+These instructions detail how to set up a minimal logserver service, including
+steps to import a database from another logserver instance.
+
+1. `git clone git@github.com:uwcirg/logserver.git`  # clone this repository
+2. `cp default.env .env`  # follow instructions within
+3. `docker compose pull`  # pulls the dependent images
+4. `docker compose up -d`  # starts the containers
+
+Test the install is functional.  Obtain a JWT and use it to POST a simple example
+document.  Correct the URL to fit.
+
+```bash
+pip install pyjwt
+export TOKEN=`python3 generate_JWT.py`
+export URL="http://unbiunium.cirg.washington.edu:3000/events"
+curl $URL -X POST -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" -d '{"event": {"name": "cmd line test"}}'
+```
```

Comment:
This is tricky to get right and might be better as a separate section/next steps